compiler/optimizer/LocalOpts.cpp

```diff
@@ -1619,32 +1619,6 @@ int32_t TR_HoistBlocks::process(TR::TreeTop *startTree, TR::TreeTop *endTree)
                // create improper regions, reducing effectiveness of opts that
                // follow.
                //
```

Comment:
Shouldn't the comment also removed?

Reply:
Just to be clear, I believe the comment at the start of the `if` block _should not_ be removed. But there's no reason to keep the one after that.